cuebot/src/main/java/com/imageworks/spcue/servant/ManageJob.java

```diff
@@ -987,5 +1101,14 @@ public JobSearchFactory getJobSearchFactory() {
     public void setJobSearchFactory(JobSearchFactory jobSearchFactory) {
         this.jobSearchFactory = jobSearchFactory;
     }
+
+    private boolean isJobFinished() {
```

Comment:
The function name could maybe be more clear. Under the default config settings it will return `false` for even finished jobs, which is the correct behavior but may be confusing for future developers. Maybe something like `isJobReadonly`?